dtfs-central-api/src/helpers/utilisation-data.helper.ts

```diff
@@ -0,0 +1,26 @@
+import { DbRequestSource, UtilisationDataEntity } from '@ukef/dtfs2-common';
+import { UtilisationReportRawCsvData } from '../types/utilisation-reports';
+import { UTILISATION_REPORT_HEADERS } from '../constants';
+
+type UtilisationDataCsvRowToSqlEntityParams = {
+  dataEntry: UtilisationReportRawCsvData;
+  requestSource: DbRequestSource;
+};
+
+const asNumberOrDefault = (maybeString: string | undefined, defaultNumber: number) => (maybeString ? Number(maybeString) : defaultNumber);
+
+export const utilisationDataCsvRowToSqlEntity = ({ dataEntry, requestSource }: UtilisationDataCsvRowToSqlEntityParams): UtilisationDataEntity =>
```

Comment:
Sorry but going to be a pain and ask for some docs on a bunch of functions, can you just check through the PR and see if there are any functions (not in tests) missing any please